docs/internals/contributing/writing-documentation.txt

```diff
@@ -210,7 +210,7 @@ Our policy for new features is:
 
 Our preferred way for marking new features is by prefacing the features'
 documentation with: "``.. versionadded:: X.Y``", followed by a mandatory
-blank line and an optional content (indented).
+blank line and an optional description (indented).
```

Comment:
"Basically, a directive consists of a name, arguments, options and content."
http://sphinx-doc.org/rest.html#directives